src/services/Warnings.js

```diff
@@ -145,6 +151,8 @@ const getWarningsZones = (warningsZone, zone, client) => {
         status: strTwitter,
         media: [fileName],
       }]);
+
+      await tClient.sendMessage(telegramKeys.chat_id, strTwitter, {});
```

Comment:
As stated by the pipeline this `await` is not valid. But in this case I think you can dismiss it since we don't need to wait for the result to proceed.

Reply:
Removed.